core/domain/collection_domain.py

```diff
@@ -199,9 +207,19 @@ def skills(self):
         return set(self.prerequisite_skills) | set(self.acquired_skills)
 
     def update_prerequisite_skills(self, prerequisite_skills):
+        """Updates prerequisite skills of collection node.
+
+        Note: Please use the add_collection_node_prerequisite_skill
```

Comment:
the update functions in collection_services still call these functions to update the actual skills, but those check for e.g. existence of skill ids before adding them.

note to self: this docstring is slightly wrong, it's now collections_services.update_collection_node_prerequisite_skills